LocalSettings.php

```diff
@@ -2894,6 +2899,17 @@
 				'edit' => true,
 			),
 		),
+		'+wikibridgewiki' => array(
+			'AbuseFilter Manager' => array(
```

Comment:
Tabbing issues and this should ideally be lowercase and one word (abusefilter-manager).